### PR TITLE
WRN-3833: Migrate internal/Picker, ProgressBar and Region unit tests to Testing Library

### PR DESCRIPTION
**Pull request by stanca-pop-lgp**
Enact-DCO-1.0-Signed-off-by: Stanca Pop stanca.pop@lgepartner.com

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Migrated unit tests from Enzyme to Testing Library for internal/Picker, ProgressBar and Region components.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRN-3833

### CODE REVIEW

File: ProgressBar/tests/ProgressBar-specs.js [hunk 1]
```diff
@@ -1,28 +1,29 @@
-import {mount} from 'enzyme';
+import '@testing-library/jest-dom';
+import {render, screen} from '@testing-library/react';
+
 import ProgressBar from '../ProgressBar';
 
 describe('ProgressBar Specs', () => {
 	test('should only show tooltip when tooltip is true', () => {
-		const progressBar = mount(
+		render(
```
Comment (alexandrumorariu) [inline ProgressBar/tests/ProgressBar-specs.js]: Please write render method on a single line(same below).

File: ProgressBar/tests/ProgressBar-specs.js [hunk 2]
```diff
@@ -1,28 +1,29 @@
...
-		expect(actual).toBe(expected);
+		expect(progressBar).toHaveClass(expected);
 	});
 
 	test('should have tooltip show progress as percentage', () => {
-		const progressBar = mount(
+		render(
 			<ProgressBar
 				tooltip
 				progress={0.6}
```
Comment (alexandrumorariu) [inline ProgressBar/tests/ProgressBar-specs.js]: Order the props in alphabetical order.

File: Region/tests/Region-specs.js [hunk 3]
```diff
@@ -7,26 +10,17 @@ describe('Region', () => {
...
 
 			test('should use aria-label when set', () => {
-				const props = {
-					'aria-label': ariaLabel,
-					title: title
-				};
-
-				const expected = ariaLabel;
-				const actual = Region.computed['aria-label'](props);
+				render(<Region title={title} aria-label={ariaLabel} />);
```
Comment (alexandrumorariu) [inline Region/tests/Region-specs.js]: Order props in alphabetical order.

File: internal/Picker/tests/Picker-specs.js [hunk 4]
```diff
@@ -164,355 +168,332 @@ describe('Picker Specs', () => {
...
-	test(
-		'should not allow keyboard decrement via down arrow keys when \'joined\' and \'horizontal\'',
-		() => {
-			const handleChange = jest.fn();
-			const picker = mount(
-				<Picker index={0} joined max={1} min={-1} onChange={handleChange} orientation="horizontal" value={0}  />
+	test('should not allow keyboard decrement via down arrow keys when \'joined\' and \'horizontal\'', () => {
+		const handleChange = jest.fn();
+		render(
+			<Picker index={0} joined max={1} min={-1} onChange={handleChange} orientation="horizontal" value={0}  />
```
Comment (alexandrumorariu) [inline internal/Picker/tests/Picker-specs.js]: There are 2 spaces after value={0}

File: internal/Picker/tests/Picker-specs.js [hunk 5]
```diff
@@ -1,35 +1,34 @@
...
-const decrement = (slider) => tap(slider.find('IconButton').last());
-const increment = (slider) => tap(slider.find('IconButton').first());
+const increment = (slider) => userEvent.click(slider.firstElementChild);
+const decrement = (slider) => userEvent.click(slider.lastElementChild);
 
 describe('Picker Specs', () => {
-
 	test('should have a default \'value\' of 0', () => {
-		const picker = mount(
+		render(
```
Comment (alexandrumorariu) [inline internal/Picker/tests/Picker-specs.js]: Please write all the render methods on a single line(as long as it has no children or it fits).

File: internal/Picker/tests/Picker-specs.js [hunk 6]
```diff
@@ -59,25 +59,24 @@ describe('Picker Specs', () => {
...
 		increment(picker);
 
-		const expected = 0;
-		const actual = handleChange.mock.calls.length;
-
-		expect(actual).toBe(expected);
+		expect(handleChange).not.toHaveBeenCalled();
 	});
 
 	test(
```
Comment (alexandrumorariu) [inline internal/Picker/tests/Picker-specs.js]: Please write test(.....) on a single line(same below).

**Review by alexandrumorariu — APPROVED** [review]
LGTM

**Review by adrian-cocoara-lgp — APPROVED** [review]
LGTM

**Pull request merged**